src/iosrv/profile.c

```diff
@@ -58,37 +67,31 @@ srv_profile_chunk_alloc(int chunk_size)
 		return NULL;
 
 	D_INIT_LIST_HEAD(&chunk->spc_chunk_list);
-
-	D_ALLOC(chunk->spc_profiles,
-		chunk_size * sizeof(*chunk->spc_profiles));
-	if (chunk->spc_profiles == NULL) {
-		D_FREE_PTR(chunk);
+	D_ALLOC(chunk->spc_chunks, chunk_size);
```

Comment:
shouldn't it be D_ALLOC(chunk->spc_chunks, chunk_size * sizeof(*chunk->spc_chunks));?

Reply:
> shouldn't it be D_ALLOC(chunk->spc_chunks, chunk_size * sizeof(*chunk->spc_chunks));?

@wangdi1 is it a bug actually? or may I missunderstood it?

Reply:
Ah, yes, you are right. I will fix it in the following patch.